corehq/apps/locations/dbaccessors.py

```diff
@@ -40,3 +40,25 @@ def get_all_users_by_location(domain, location_id):
         reduce=False,
     )
     return (CouchUser.wrap_correctly(res['doc']) for res in results)
+
+
+def users_have_locations(domain):
+    from corehq.apps.users.models import CouchUser
+    return bool(CouchUser.get_db().view(
+        'locations/users_by_location_id',
+        startkey=[domain],
+        endkey=[domain, {}],
+        reduce=True,
+    ).one())
+
+
+def get_all_users_with_locations(domain):
```

Comment:
picky/silly but I might name this `get_all_users_with_locations_assigned` or something since when I first read the function I thought it would be returning users and locations together somehow